inst/include/population_dynamics/growth/functors/ewaa.hpp

```diff
@@ -43,5 +43,5 @@ struct EWAAgrowth : public GrowthBase<Type> {
     return ret;
   }
 };
-}  // namespace fims
+}  // namespace fims_popdy
```

Comment:
This is cool. I wasn't aware that the Clang Format GitHub Actions workflow also corrects documentation.